### PR TITLE
chore: Update README.md and Added CONTRIBUTING.md

### LINKED ISSUE #63: ReadMe file not descriptive enough
**Actual Behaviour**
ReadMe file is not that descriptive about the app. No functioning of the app is described clearly.

**Expected Behaviour**
It should briefly explain the working as well as purpose of the app. Also it should tell the contribution process.This will definitely help those beginner contributors.

**Steps to reproduce it**
1.) By explaining even a little about the working of app, features (developed or to be developed).
2.) By attaching screen shots of the working app.

**Would you like to work on the issue?**
I personally think that creator should update the readme for the better understanding of an

### PR DESCRIPTION
**Pull request by vishal-sehgal**
Fixes #63 
**Changes**:
Update the README.md and added screenshots for a better understanding of this project.

**Checklist**:
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications are done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

### CODE REVIEW

**Comment by vishal-sehgal** [comment]
@iamareebjamal Can you please review this? I can't believe after 10th attempt my checks are successfully passed.

**Comment by iamareebjamal** [comment]
Peer reviews first

**Comment by vishal-sehgal** [comment]
@iamareebjamal Yes, sure
@suresh-gandhi Can you please provide feedback for this?

**Review by suresh-gandhi — COMMENTED** [review]
@CoderVishalSehgal I think this should be a part of "How to install" section in Readme. Readme typically includes introducing the community about the project, its purpose, basic installation, requirements, etc. Contributing file, on the other hand, is meant to assist readme by setting up the best possible contributing guidelines for the newcomers especially. For reference take a look [here](https://gist.github.com/PurpleBooth/b24679402957c63ec426).

**Comment by vishal-sehgal** [comment]
@suresh-gandhi I mentioned all the details already, can you please elaborate more what do you want me to change in **README.md**, Thanks.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+Click the green button “Create pull request”. Give a succinct and informative title, in the comment field give a short explanation of the changes and click the green button “Create pull request” again.
+
+#### Pulling others’ changes
+Before you make further changes to the repository, you should check that your version is up to date relative to FOSSASIA’s version.
+
+Go into the directory for the project and type:
+
+`$ git checkout development`
+`$ git pull upstream development --rebase`
```
Comment (suresh-gandhi) [inline CONTRIBUTING.md:62]: These all should be a part of How to install(or stay updated, essentially) subsection in readme. I mean these are all general though correctly written things but shouldn't be a part of contributing file. Contributing file is typically focussed on best practices, coding standards, some conventions followed by the organization, any particular coding style invariant to be kept in mind, etc. You can take a look at a sample contributing file [here](https://gist.github.com/PurpleBooth/b24679402957c63ec426).

**Comment by Pipe-Runner** [comment]
@CoderVishalSehgal Squash ur commits before dropping a PR.

**Review by suresh-gandhi — COMMENTED** [review]
@CoderVishalSehgal Could you please update the PR with the suggested changes.

**Comment by vishal-sehgal** [comment]
@AakashMallik yes sure buddy

**Comment by vishal-sehgal** [comment]
@suresh-gandhi I am on bed rest from last week :( I will try to update it by tonight.

**Comment by suresh-gandhi** [comment]
@CoderVishalSehgal Oh Sorry to hear that. No worries, take your time. Get well soon.

**Review by mariobehling — APPROVED** [review]

**Pull request merged**